tests/results/DiffTest.py

```diff
@@ -279,3 +279,17 @@ def test_delete(self):
         self.uut.delete = True
         self.assertEqual(self.uut.modified, [])
         self.uut.delete = False
+
+    def test_diff_creation(self):
+        uut = Diff(create='create_test.py')
+        self.assertEqual(uut.create, 'create_test.py')
+        self.assertEqual(uut._file, [])
+        uut.add_lines(0, ['line2', 'line3'])
+        self.assertEqual(uut.modified, ['line2', 'line3'])
```

Comment:
It should print the file you change with it's filename, like said in the issue you should print a remaining from `/dev/null` to the new `create`-name or so. So imo this needs a test^^